test/compress/dead-code.js

```diff
@@ -256,3 +256,16 @@ try_catch_finally: {
         "1",
     ]
 }
+
+accessor: {
+    options = {
+        side_effects: true,
+    }
+    input: {
+        ({
+            get a() {},
+            b: 1
+        });
+    }
+    expect: {}
```

Comment:
I don't understand this test result. Oh I see - it's dropping a side effect free statement?

Could you please add a setter in the same object literal just for coverage sake?
```
$ echo '({set a(){}, set a(v){this.b = 2}, b:1});' | bin/uglifyjs -c side_effects=0
({set a(){},set a(v){this.b=2},b:1});

$ echo '({set a(){}, set a(v){this.b = 2}, b:1});' | bin/uglifyjs -c side_effects=1
# nothing
```